foreign_cc/private/cc_toolchain_util.bzl

```diff
@@ -368,14 +368,25 @@ def get_flags_info(ctx, link_output_file = None):
         ),
     )
     return CxxFlagsInfo(
-        cc = _add_if_needed(flags.cc, copts),
-        cxx = _add_if_needed(flags.cxx, cxxopts),
-        cxx_linker_shared = _add_if_needed(flags.cxx_linker_shared, linkopts),
-        cxx_linker_static = flags.cxx_linker_static,
-        cxx_linker_executable = _add_if_needed(flags.cxx_linker_executable, linkopts),
-        assemble = _add_if_needed(flags.assemble, copts),
+        cc = _convert_flags(cc_toolchain_.compiler, _add_if_needed(flags.cc, copts)),
+        cxx = _convert_flags(cc_toolchain_.compiler, _add_if_needed(flags.cxx, cxxopts)),
+        cxx_linker_shared = _convert_flags(cc_toolchain_.compiler, _add_if_needed(flags.cxx_linker_shared, linkopts)),
+        cxx_linker_static = _convert_flags(cc_toolchain_.compiler, flags.cxx_linker_static),
+        cxx_linker_executable = _convert_flags(cc_toolchain_.compiler, _add_if_needed(flags.cxx_linker_executable, linkopts)),
+        assemble = _convert_flags(cc_toolchain_.compiler, _add_if_needed(flags.assemble, copts)),
     )
 
+def _convert_flags(compiler, flags):
```

Comment:
Can you make this an isolated PR? Seems like a great change 😄

Reply:
Please see #731